### ISSUE TITLE
[Nintendo Switch] save in game cannot work. not enough memory space.

### ISSUE THREAD

**Issue by lencil** [hrydgard/ppsspp#15704]
### Game or games this happens in

all

### What area of the game / PPSSPP

all

### What should happen

game cannot save. In game prompt something like "not enough space".

### Platform

Switch

### Mobile phone model or graphics card

switch

### PPSSPP version affected

latest

### Last working version

no known

### Graphics backend (3D API)

OpenGL / GLES

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [ ] Try resetting settings or older versions and include if the issue is related.
- [ ] Try without any cheats and without loading any save states.
- [ ] Include logs or screenshots of issue.

**Comment by hrydgard** [comment]
can you try these instructions from m4xw and see what you get?

```
nxlink -s -a <listen ip> /path/to/nro [-- -optional -startup -args] 
```
> then press Y in hbmenu and it will upload the nro and run

Or you really are out of space...

**Comment by lencil** [comment]
> 

the space is quite enough.
but I don't know how to use nxlink, and upload log. Any detail guide for me to do this?

**Comment by lencil** [comment]
I found out how it occurs. When setup retroarch to use savefiles in the same folder as roms the save function cannot work well.

**Comment by unknownbrackets** [comment]
This issue is being closed as a duplicate of #15437.

This is an issue tracker - there's a [forum if you want to discuss things](http://forums.ppsspp.org/).  You wouldn't add another "bread" item to a grocery list every time you thought about it, right?  We do the same here - we already have "bread" (this issue) on our list.

By keeping the list clean, we can also make sure all the information a developer needs to fix the issue is at their finger tips.  When there are multiple issues, information gets lost and it becomes hard to fix things, so they are fixed more slowly.  Duplicates won't make fixes appear faster.

And remember - PPSSPP is open source.  Try to find *new and useful information*, and add it to the original issue.  If you can't wait any longer, muster up your courage and try fixing it yourself.  PPSSPP gets better the more people contribute.  You could be that amazing person who solves this problem for everyone.

-[Unknown]
